xml/ftp_banners.xml

```diff
@@ -617,6 +617,20 @@ more text</example>
     <param pos="0" name="hw.device" value="Multifunction Device"/>
     <param pos="1" name="hw.product"/>
   </fingerprint>
+  <fingerprint pattern="Xerox WorkCentre (\d+).*$" certainty="1.0">
+    <description>Xerox WorkCentre</description>
+    <example hw.version="6605">Xerox WorkCentre 6605DN</example>
```

Comment:
`6605DN` should be the `hw.product` since it's a Xerox WorkCentre model 6605DN.  `hw.version` would be more in line with the version of the hardware released for a particular model. 

Similar feedback below.